### PR TITLE
[NetAppFiles]Added Revert cmdlet, update to 2020-02-01 apiversion

### PR DESCRIPTION
**Pull request by audunn**
## Description
* Added SnapshotId parameter to New-NetAppFilesVolume to create volume from a snapshot
* Added Revert-AzNetAppFilesVolume to revert a volume to one of its snapshots
* FileSystemId removed from Snapshot
* Added BackupOperators to ActiveDirectory, list of users to add to Backup Operator active directory group
* Added Status, OrganizationalUnit and Site properties to ActiveDirectory
* Added SnapshotDirectoryVisible to Volume, If enabled (true) the volume will contain a read-only .snapshot directory which provides access to each of the volume's snapshots (default to true)
* Added Snapshot to Volume DataProtection

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/NetAppFiles/NetAppFiles.Test/NetAppFiles.Test.csproj [hunk 1]
```diff
@@ -11,8 +11,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.NetApp" Version="1.4.0" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="20.0.2-preview" />
```
Comment (msJinLei) [inline src/NetAppFiles/NetAppFiles.Test/NetAppFiles.Test.csproj:15]: Please resolve the conflicts here.

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/NetAppFiles/NetAppFiles/ChangeLog.md [hunk 2]
```diff
@@ -19,6 +19,16 @@
 -->
 ## Upcoming Release
 
+## Version 0.1.6
+
```
Comment (msJinLei) [inline src/NetAppFiles/NetAppFiles/ChangeLog.md:23]: can you remove the empty line?

**Review by msJinLei — CHANGES_REQUESTED** [review]
Basically OK. Please resolve the conflicts and remove the empty line.

File: src/NetAppFiles/NetAppFiles.Test/NetAppFiles.Test.csproj (hunk 1 above)
Comment (audunn) [reply src/NetAppFiles/NetAppFiles.Test/NetAppFiles.Test.csproj:15]: resolved

File: src/NetAppFiles/NetAppFiles/ChangeLog.md (hunk 2 above)
Comment (audunn) [reply src/NetAppFiles/NetAppFiles/ChangeLog.md]: removed

**Review by msJinLei — APPROVED** [review]

**Pull request merged**